src/App.js

```diff
@@ -29,10 +29,7 @@ export default class App extends Component {
       })
       .then(() => this.getFilters('platforms'))
       .then(() => this.getFilters('genres'))
-      .catch(error => {
-        throw new Error(error)
-      })
-    fetch('http://whateverly-datasets.herokuapp.com/api/v1/genres1811')
+      .then(() => fetch('http://whateverly-datasets.herokuapp.com/api/v1/genres1811'))
```

Comment:
Good thinking slimming this down a few lines, but do we need another catch error?